vlm/src/test/scala/geotrellis/contrib/vlm/RasterSourceRDDSpec.scala

```diff
@@ -146,26 +146,33 @@ class RasterSourceRDDSpec extends FunSpec with TestEnvironment with BetterRaster
 
     describe("GDALRasterSource") {
       val expectedFilePath = s"${new File("").getAbsolutePath()}/src/test/resources/img/aspect-tiled-near-merc-rdd.tif"
-      val reprojectedExpectedRDDGDAL: MultibandTileLayerRDD[SpatialKey] =
-        RasterSourceRDD(GDALRasterSource(expectedFilePath), layout)
 
-      val rasterSource = GDALRasterSource(filePath)
       it("should reproduce tileToLayout") {
+        val rasterSource = GDALRasterSource(filePath)
```

Comment:
This isn't an issue so much as a question, but will we need to instantiate a `GDALRasterSource` for each test from now one?

Reply:
Yes, just to be sure that we clean links as expected and in a proper moment, if we'll use the same resource for multiple tests, than there would appear a question, when do we want to close it?